Firmware_Bootstrap_Loader/README.md

```diff
@@ -0,0 +1,78 @@
+# Progaming Faraday - Bootstrap Loader (Windows)
```

Comment:
I would try to style this like my [APRS](https://github.com/FaradayRF/Faraday-Software/tree/master/Applications/APRS) or [Telemetry](https://github.com/FaradayRF/Faraday-Software/tree/master/Applications/Telemetry) application readme.md files. You're 90% of the way there. Styling of configuration options and step by step items really help. You probably don't need as many images. That kind of blows the page up.